### PR TITLE
Fall Banquet 2017 Update

### PR DESCRIPTION
**Pull request by amicableskeptic**
Adding post on Fall Dinner 2017 and doing a bunch of other small tweaks. Killing Archives and Spotlight on Fencers sections. Adding new email archives section.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -123,6 +128,20 @@ <h2 class="title-margin-bottom">News &amp; Events</h2>
                
                <div class="eight columns">
                    
+                   <h4>Second Annual Cornell Fencing Association Fall Dinner</h4>
+                   <img class="news-img" src="images/2017FallDinner.jpg" alt="Coaches with the 1967 Champions" />
```
Comment (NickHeiner) [inline app/index.html:132]: Looks like you did not check in these images.

File: app/index.html [hunk 2]
```diff
@@ -215,13 +191,11 @@ <h2 class="title-margin-bottom">News &amp; Events</h2>
                     <a href="http://cornellfencing.com/schedule/currentschedule" target="_blank">Men's Schedule</a> | 
                     <a href="http://www.cornellbigred.com/schedule.aspx?path=fencing&" target="_blank">Women's Schedule</a>
                    <p>
-                   <h5>Cornell Fencing Fall Banquet</h5>
+                   <h5>Cornell Fencing Spring Banquet and Alumni Tournament</h5>
                    <p>
-                       Our second annual fall fencing banquet at the Cornell Club in NYC. We hope you'll come join us to connect with your old fencing friends, make a few new ones and Honor the 1967 Women's National Championship Team in New York City!!
+                       Mark your calendars now! The annual Cornell Fencing Spring Banquet has been scheduled for April 14, 2018. This years banquet we will be honoring Raoul Sudre in addition to fencing, catching up with old friends, and maybe making some new ones.
```
Comment (NickHeiner) [inline app/index.html]: > This years banquet we will be

This is missing at least an apostrophe. How about:

> This year's banquet, we will be

File: app/index.html [hunk 3]
```diff
@@ -234,12 +208,11 @@ <h2 class="title-margin-bottom">News &amp; Events</h2>
...
             <h2 class="title-margin-bottom">Media</h2>
-               <h2>Archives</h2>
-               <p>We look forward to our Archives section, coming soon!  You'll be able to browse photos of many generations of Cornell Fencing.<br>
-               Follow us on social media!
-               </p>
-
-               <h2>Social Media</h2>
+              <h3>Email Newsletter Archives</h3>
+              
+              <p>Cornell Fencing sends out regular email updates about how the program is doing. You can read through the e-mails that have been sent and subscribe to future ones by clicking <a href="http://us13.campaign-archive.com/home/?u=6d228c291b3baeafb9ec6c712&id=812a3b5c0a">this link</a>.</p>
```
Comment (NickHeiner) [inline app/index.html]: Click [here]()-style links tend to be worse for usability and accessibility. It's better to make the link be named for the thing that it's linking to. In this case, we could do:

> You can [read through the emails that have been sent and sign up for future ones]()

Or:

> You can read through the emails that have been sent and sign up for future ones on [our mailing list]().

**Review by NickHeiner — CHANGES_REQUESTED** [review]
Just a few changes, then this will be ready to go.

**Comment by amicableskeptic** [comment]
Check out my latest changes and deploy if everythign looks good.

**Pull request merged**